javaparser-core/src/main/java/com/github/javaparser/javadoc/description/JavadocInlineTag.java

```diff
@@ -87,7 +87,7 @@ public JavadocInlineTag(Type type, String content) {
 
     @Override
     public String toText() {
-        throw new UnsupportedOperationException();
+        return "{@" + type.keyword + this.content;
```

Comment:
Yes, probably I should have done that. I think it is wrong the closing brace is part of the content